Dockerfile

```diff
@@ -8,13 +8,34 @@ RUN curl -sL https://deb.nodesource.com/setup_4.x | bash -
 RUN apt-get update && apt-get install -y nodejs
 RUN npm install -g npm
 
-# copy in the pip-service code
-WORKDIR /pip
-COPY . .
+# download WOF data
+RUN mkdir /whosonfirst
+
+# copy pelias config
+ADD './docker/pelias.json' '/code/pelias.json'
+ENV PELIAS_CONFIG '/code/pelias.json'
+
+# clone repo
+RUN git clone https://github.com/pelias/whosonfirst.git /code/pelias/whosonfirst
+
+# change working dir
+WORKDIR /code/pelias/whosonfirst
+
+# install npm dependencies
+RUN git checkout filtered-download
```

Comment:
This won't be needed once the changes from filtered-download branch is merged with master?